modules/javafx.graphics/src/main/java/com/sun/glass/ui/gtk/screencast/TokenStorage.java

```diff
@@ -109,26 +111,33 @@ private static Path setupPath() {
             return null;
         }
 
-        Path path = Path.of(userHome, REL_NAME);
+        Path secondaryPath = Path.of(userHome, REL_NAME_SECONDARY);
+
+        // use secondary path only if it already exists and writable
+        Path path = Files.isWritable(secondaryPath)
```

Comment:
If both exist and are writable, then wouldn't you want to default to the primary path? If so, then I think the test should be something like `Files.isWritable(secondaryPath) && !Files.isWritable(primaryPath)`.

Reply:
Sure, it sounds reasonable, updated.